CHANGELOG.md

```diff
@@ -4,6 +4,10 @@
 
 <a name="breaking_changes_1.5.0">[Breaking Changes:](#breaking_changes_1.5.0)</a>
 
+<a name="1_4_0_drop_node_10_support"></a>
+- [[repo]](#1_4_0_drop_node_10_support) support for `Node 10` is dropped. [#8290](https://github.com/eclipse-theia/theia/pull/8290)
+  - From now on, you Node.js `12.x` is required to build Theia from sources.\
```

Comment:
remove `you`?

Reply:
"to build Theia from sources"  - if one builds from npm packages, is it different? I guess eventually node 12 will be needed for that as well? If so we could remove this bit?